.github/workflows/code_testing.yaml

```diff
@@ -19,8 +19,6 @@ jobs:
           - os: ubuntu-22.04
             compiler: clang-19
 
-          - os: ubuntu-22.04
-            compiler: gcc-12
           - os: ubuntu-22.04
             compiler: gcc-13
```

Comment:
Please add gcc-14 and 15

Reply:
gcc-15 seems not yet supported on ubuntu  24.04